analytics.js

```diff
@@ -247,6 +261,9 @@ class Analytics {
           "[Analytics] init :: trying to initialize integration name:: ",
           intg.name
         );
+        leaveBreadcrumb(
+          `[Analytics] init :: trying to initialize integration name:: ${intg.name}`
+        );
```

Comment:
Please do not repeat the error messages.

Reply:
done